index.bs

```diff
@@ -326,23 +326,8 @@ spec: workers; urlPrefix: https://www.w3.org/TR/workers/#
             abort these steps.
           </li>
           <li>
-            If none of the following is true, return <code>false</code> and
-            abort these steps:
-            <ul>
-              <li>
-                Applying the <a>rules for parsing floating-point number
-                values</a> to <var>configuration</var>'s
-                {{VideoConfiguration/framerate}} results in a number that is
-                finite and greater than 0.
-              </li>
-              <li>
-                <var>configuration</var>'s {{VideoConfiguration/framerate}}
-                contains one occurrence of U+002F SLASH character (/) and the
-                substrings before and after this character, when applying the
-                <a>rules for parsing floating-point number values</a> results in
-                a number that is finite and greater than 0.
-              </li>
-            </ul>
+            If {{VideoConfiguration/framerate}} is not finite or is not greater
```

Comment:
I take as implicitly banning NaN as well. Just calling it out in case that needed special mention.

Reply:
Yeah, it should.